src/components/table/table-cell/bl-table-cell.ts

```diff
@@ -0,0 +1,99 @@
+import { html, LitElement, TemplateResult } from "lit";
+import { customElement, property } from "lit/decorators.js";
+import { CSSResultGroup } from "lit/development";
+import "element-internals-polyfill";
+import "../../checkbox-group/checkbox/bl-checkbox";
+import style from "../table-cell/bl-table-cell.css";
+import BlTableRow, { blTableRowTag } from "../table-row/bl-table-row";
+
+export const blTableCellTag = "bl-table-cell";
+/**
+ * @tag bl-table-cell
+ * @summary Baklava Table component
+ */
+@customElement(blTableCellTag)
+export default class BlTableCell extends LitElement {
+  static get styles(): CSSResultGroup {
+    return [style];
+  }
+
+  /**
+   * Disable selection
+   */
+  @property({ type: Boolean, reflect: true, attribute: "disable-selection" })
```

Comment:
If the intended functionality aligns with disabling, we may want to contemplate renaming this attribute to `disabled`.